src/methods/collections/index.ts

```diff
@@ -0,0 +1,52 @@
+import { handleFeedResponse } from '../../helpers/feed';
+import { compactDefined } from '../../helpers/fp';
+import * as Query from '../../helpers/query';
+import { createRequestHandler, createRequestHandlerOptional } from '../../helpers/request';
+import { castResponse } from '../../helpers/response';
+import { OrientationParam, PaginationParams } from '../../types/request';
+
+type CollectionId = {
+  collectionId: string;
+};
+
+const COLLECTIONS_PATH_PREFIX = '/collections';
+
+export const getPhotos = {
+  handleRequest: createRequestHandler(
+    ({
+      collectionId,
+      orientation,
+      ...paginationParams
+    }: CollectionId & PaginationParams & OrientationParam) => ({
+      pathname: `${COLLECTIONS_PATH_PREFIX}/${collectionId}/photos`,
+      query: compactDefined({ ...Query.getFeedParams(paginationParams), orientation }),
```

Comment:
Do we need to use `compactDefined` here? I thought you decided to do that inside `createRequestHandler`?

Also applies to other places in this PR.

Reply:
Oh sorry. I ended up tightening the `Query` type to not allow any non-values after all, which required me to add `compactDefined` wherever it was needed in methods. We no longer need to compact in `createRequestHandler`, so I removed it from there just now.